aim/sdk/num_utils.py

```diff
@@ -86,23 +98,31 @@ def is_number(value):
     if is_pytorch_tensor(value):
         return True
 
+    if is_tensorflow_tensor(value):
+        return True
+
     return False
 
 
 def convert_to_py_number(value) -> object:
     """
     Converts numpy objects or tensors to python number types
     """
-    if isinstance(value, float):
-        return float(value)
+    if is_py_number(value):
+        return value
 
-    if isinstance(value, int):
-        return int(value)
+    if is_numpy_array(value):
+        if value.shape != (1,):
+            raise ValueError('Not a scalar numpy array')
+        return value.item()
 
     if is_numpy_number(value):
         return value.item()
 
     if is_pytorch_tensor(value):
         return value.item()
 
+    if is_tensorflow_tensor(value):
+        return value.eval()
```

Comment:
I guess we need to call `item()` on top, since `eval()` will return `numpy.ndarray`